code/modules/mob/living/carbon/human/life.dm

```diff
@@ -847,7 +843,7 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 
 	if(species && species.flags & NO_INTORGANS) return
 
-	if(!(species.flags & IS_SYNTHETIC)) handle_trace_chems()
+	handle_trace_chems()

```

Comment:
This proc handles tracing chemicals through random organs, which seems to be mostly used for autopsy data. It's use for IPCs is somewhat debatable for that reason.